src/js/libcs/OpImageDrawing.js

```diff
@@ -385,3 +385,169 @@ evaluator.cameravideo$0 = function() {
     });
     return img;
 };
+
+var tmpcanvas; //temporary unvisible canvas.
+/**
+ * reads a rectangular block of pixels from the upper left corner.
+ * The colors are representent as a 4 component RGBA vector with entries in [0,1]
+ */
+function readPixelsIndirection(img, x, y, width, height) {
+    var res = [];
+    if (img.readPixels) {
+        res = img.readPixels(x, y, width, height);
+    } else { //use canvas-approach
+        var data;
+        if (img.img.getContext) { //img is a canvas
+            var ctx = img.img.getContext('2d');
+            data = ctx.getImageData(x, y, width, height).data;
+        } else { //copy corresponding subimage of img.img to temporary canvas
+            if (!tmpcanvas) {
+                //creating tmpcanvas only once increases the running time
+                tmpcanvas = /** @type {HTMLCanvasElement} */ (document.createElement("canvas"));
+                tmpcanvas.style.display = "none";
+                document.body.appendChild(tmpcanvas);
+            }
+            tmpcanvas.width = width;
+            tmpcanvas.height = height;
+
+            var tmpctx = tmpcanvas.getContext('2d');
+            tmpctx.drawImage(img.img, x, y, width, height, 0, 0, width, height);
+            data = tmpctx.getImageData(0, 0, width, height).data;
+        }
+        for (var i in data) res.push(data[i] / 255);
+    }
+    return res;
+}
+
+/**
+ * imagergba(‹image›,x,y) implements imagergb(‹imagename›,x,y) from Cinderella, i.e.
+ * returns a 4 component vector ranging from (0-255, 0-255, 0-255, 0-1)
+ */
+evaluator.imagergba$3 = function(args, modifs) {
+    var img = imageFromValue(evaluateAndVal(args[0]));
+    var x = evaluateAndVal(args[1]);
+    var y = evaluateAndVal(args[2]);
+
+    if (!img || x.ctype !== 'number' || y.ctype !== 'number') return nada;
+
+    x = Math.round(x.value.real);
+    y = Math.round(y.value.real);
+    if (isNaN(x) || isNaN(y)) return nada;
+
+    var rgba = readPixelsIndirection(img, x, y, 1, 1);
+    return List.realVector([rgba[0] * 255, rgba[1] * 255, rgba[2] * 255, rgba[3]]);
+};
+
+evaluator.imagergb$3 = evaluator.imagergba$3; //According to reference
+
+/**
+ * imagergba(<point1>, <point2>, ‹image›, <point3>) returns the color at the coordinate
+ * <point3> assuming that the left/right lower corner is <point1>/<point2> resp.
+ */
```

Comment:
Please add this to the reference manual, and remove the `not implemented yet` notice for the Cinderella-compatible function.